Fix missing altcoins tab

Kukks opened the pull request.

NicolasDorier commented on BTCPayServer/Views/Shared/Bitcoin_Lightning_LikeMethodCheckout.cshtml:
```diff
@@ -151,8 +151,17 @@
         </div>
-        
-        <div id="tabsSlider" class="payment-tabs__slider"  v-bind:class="['slide-'+currentTab]"></div>
+        @if (Model.CoinSwitchEnabled)
```
I thought we removed CoinSwitch?

Kukks commented:
No we removed changelly

On Thu, 29 Oct 2020, 04:53 Nicolas Dorier, <notifications@github.com> wrote:

> *@NicolasDorier* commented on this pull request.
> ------------------------------
>
> In BTCPayServer/Views/Shared/Bitcoin_Lightning_LikeMethodCheckout.cshtml
> <https://github.com/btcpayserver/btcpayserver/pull/2027#discussion_r513935569>
> :
>
> > @@ -151,8 +151,17 @@
>          <div class="payment-tabs__tab" id="copy-tab" v-on:click="switchTab('copy')"  v-bind:class="{ 'active': currentTab == 'copy'}" >
>              <span>{{$t("Copy")}}</span>
>          </div>
> -
> -        <div id="tabsSlider" class="payment-tabs__slider"  v-bind:class="['slide-'+currentTab]"></div>
> +        @if (Model.CoinSwitchEnabled)
>
> I thought we removed CoinSwitch?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/btcpayserver/btcpayserver/pull/2027#pullrequestreview-519315202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAN357WQIJX433W44GQMQATSNDRLFANCNFSM4TCTM5EQ>
> .
>

NicolasDorier commented:
Could be a plugin? :D

Kukks commented:
> Could be a plugin? :D

For sure!
